### PR TITLE
crdoco

### PR DESCRIPTION
**Pull request by CharlesAuthier**

### CODE REVIEW

File: article/_posts/2019-07-17-CrDoCo.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Introduction
```
Comment (lemairecarl) [inline article/_posts/2019-07-17-CrDoCo.md]: yaml header is missing

File: article/_posts/2019-07-17-CrDoCo.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# Results
+
+![](/article/images/CrDoCo/results.png)
+
+![](/article/images/CrDoCo/otherresults.png)
+
+# Limitations
+
+This method is memory-intensive as the training involves multiple networks at the same time.
+They suggest to approaches this issue include adopting partial sharing on the two task networks, share the last few layers of the two task networks, and sharing the encoders in the image translation network.
```
Comment (lemairecarl) [inline article/_posts/2019-07-17-CrDoCo.md]: Their suggested approaches to this issue

File: article/_posts/2019-07-17-CrDoCo.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    venue:   "Computer Vision and Pattern Recognition (CVPR), 2019, pp. 1791-1800"
+pdf: "http://openaccess.thecvf.com/content_CVPR_2019/papers/Chen_CrDoCo_Pixel-Level_Domain_Transfer_With_Cross-Domain_Consistency_CVPR_2019_paper.pdf"
+---
+
+
+# Introduction
+
+![](/article/images/CrDoCo/dataset.png)
+
+This paper present a novel pixel-wise adversarial domain adaptation algorithm by leveraging image-to-image translation methods for data augmentation.
```
Comment (jhlegarreta) [inline article/_posts/2019-07-17-CrDoCo.md]: presents

File: article/_posts/2019-07-17-CrDoCo.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+![](/article/images/CrDoCo/method.png)
+
+The proposed network above takes an image $$I_S$$ from the source domain and another image $$I_T$$ from the target domain as inputs.
+They use the image translation network to obtain the corresponding translated images $$I_S \rightarrow T = G_S \rightarrow T (I_S)$$ (in the target domain) and $$I_T \rightarrow S = G_T \rightarrow S(I_T)$$ (in the source domain).
+Then pass $$I_S$$ and $$I_T \rightarrow S$$ to $$F_S$$, $$I_T$$ and $$I_S \rightarrow T$$ to $$F_T$$ to obtain their task predictions.
+
+## Cross-domain consistency Loss
+
+For semantic segmentation task, they use the bi-directional KL divergence loss and define the cross-domain consistency loss for semantic segmentation.
```
Comment (jhlegarreta) [inline article/_posts/2019-07-17-CrDoCo.md]: For semantic segmentation tasks, (...)

File: article/_posts/2019-07-17-CrDoCo.md (hunk 3 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-17-CrDoCo.md]: Done in e40794d.

File: article/_posts/2019-07-17-CrDoCo.md (hunk 4 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-17-CrDoCo.md]: Done in e40794d.

**Pull request merged**